### PR TITLE
product

### PR DESCRIPTION
**Pull request by rekhakilari**
- [DEMO LINK](https://rekahkilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.css [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  background: #fff;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  display: block;
+
+  margin: 0 3px 40px;
```
Comment (meforoma) [inline src/styles/blocks/card.css:15]: better use auto for vertical paddings for auto-centering
```suggestion
  margin: 0 auto 40px;
```

File: src/styles/blocks/card.css [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.card__product-code {
+  display: flex;
+
+  line-height: 14px;
+
+  color: #616070;
+
+  margin: 0 0 16px;
```
Comment (meforoma) [inline src/styles/blocks/card.css:37]:
```suggestion
  margin-bottom: 16px;
```

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: flex-end;
```
Comment (meforoma) [inline src/styles/blocks/card.css:43]: not sure why flex-end - generally doesn't really matter in this case
yet center is a better option for centered elements

**Review by meforoma — CHANGES_REQUESTED** [review]
@rekhakilari
- please, don't commit file `package-lock.json`
- your deploy doesn't show up - please fix it: https://rekahkilari.github.io/layout_product-cards-en/
- fix comments along the code

File: src/styles/blocks/card.css (hunk 3 above)
Comment (rekhakilari) [reply src/styles/blocks/card.css:43]: changes are done

**Comment by rekhakilari** [comment]
- [DEMO LINK](https://rekahkilari.github.io/layout_product-cards-en/)
- [TEST REPORT LINK](https://rekhakilari.github.io/layout_product-cards-en/report/html_report/)

**Comment by rekhakilari** [comment]
Changes done check it once

**Pull request closed without merging**